### PR TITLE
Screens and routing

### PR DESCRIPTION
**Pull request by gopalbbharadva**
#### Pages
- Home
- Rules
- Quizzes
- Question
- Result
- sign in 
- signup

Live link : https://626faab1e553af1aa4357c79--stellar-malasada-714a70.netlify.app/

### CODE REVIEW

File: src/components/Inputs/TextInput.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
```
Comment (msdeshmukh009) [inline src/components/Inputs/TextInput.tsx]: This import is not necessary now. You can skip this line.

File: src/utils/AuthValidation.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+  //   if (!email.match(emailRegex)) err.email = "Enter valid email address";
+
+  //   if (!password.match(passwordFormat)) err.password = "Enter valid password";
+
+  //   if (confirmPassword !== password)
+  //     err.confirmPassword = `Confirm password doesn't match`;
+
+  //   return err;
+};
```
Comment (msdeshmukh009) [inline src/utils/AuthValidation.ts]: You can remove this comment if not needed.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good. Nice work!!

**Review by srishti-maurya — APPROVED** [review]
Code is clean and readable, UI looks good

File: src/components/Inputs/TextInput.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { LoginInputProps } from "../../types/Auth.types";
+
+export const TextInput = ({
+  changeHandler,
```
Comment (poornima-alugubelly) [inline src/components/Inputs/TextInput.tsx:4]: nice abstraction on text input

File: src/types/Auth.types.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+export type LoginErrorsProps = {
```
Comment (poornima-alugubelly) [inline src/types/Auth.types.ts:1]: i like how you have used separate files for type definitions

**Review by poornima-alugubelly — APPROVED** [review]
I like how you have properly defined types everywhere I did not see a single `any`  type , good work 👍

**Comment by gopalbbharadva** [comment]
> I like how you have properly defined types everywhere I did not see a single `any` type , good work +1

thanks

**Pull request merged**